<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1871</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Green Monkey</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="transformsGroupBox">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>170</y>
      <width>240</width>
      <height>500</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string extracomment="Transformation 2"/>
    </property>
    <property name="whatsThis">
     <string extracomment="Transform2"/>
    </property>
    <property name="title">
     <string>Transformations</string>
    </property>
    <widget class="QPushButton" name="edgesButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Edges, Sobel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brightnessButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>You can change brightness with left slider and blur with rigt one.</string>
     </property>
     <property name="whatsThis">
      <string>You can change brightness with left slider and blur with rigt one.</string>
     </property>
     <property name="text">
      <string>Brightness and Blur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getSizeButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>get Size</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="paramsGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>91</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <widget class="QSlider" name="leftSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>20</width>
        <height>240</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Adaptive Threshold
blockSize – Size of a pixel neighborhood that is used to calculate a threshold value for the pixel: 3, 5, 7, and so on.</string>
      </property>
      <property name="whatsThis">
       <string>Adaptive Threshold
blockSize – Size of a pixel neighborhood that is used to calculate a threshold value for the pixel: 3, 5, 7, and so on.</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
     <widget class="QSlider" name="rightSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>20</width>
        <height>240</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Adaptive Threshold
C – Constant subtracted from the mean or weighted mean (see the details below). Normally, it is positive but may be zero or negative as well.</string>
      </property>
      <property name="whatsThis">
       <string>Adaptive Threshold
C – Constant subtracted from the mean or weighted mean (see the details below). Normally, it is positive but may be zero or negative as well.</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
     <widget class="QLabel" name="leftLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>20</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="rightLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>20</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="adaptiveThresholdButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adaptive Threshold</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gray2colSelButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Gray scale 0..255 to color with selected method only.
Image is converted to gray and finally to color.</string>
     </property>
     <property name="whatsThis">
      <string>Gray scale 0..255 to color with selected method only.
Image is converted to gray and  and finally to color.</string>
     </property>
     <property name="text">
      <string>Gray2Color Sel.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gray2colAllButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Gray scale 0..255 to color for all available methods.
Image resized as per scale window and then  is converted to gray and finally to color.</string>
     </property>
     <property name="whatsThis">
      <string>Gray scale 0..255 to color for all available methods.
Image resized as per scale window and then  is converted to gray and finally to color.</string>
     </property>
     <property name="text">
      <string>Gray2Color All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fftButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>FFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dftButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>DFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gaborButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Applies Gabor Filter</string>
     </property>
     <property name="whatsThis">
      <string>Applies Gabor Filter</string>
     </property>
     <property name="text">
      <string>Gabor Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="differenceButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Difference</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RGB2GrayButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB to Gray</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="invertedCheckBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Inverted Image</string>
     </property>
    </widget>
    <widget class="QDial" name="angleDial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>81</width>
       <height>64</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>GABOR Filter - angle 1..4 ~ 1*np.pi/angle</string>
     </property>
     <property name="whatsThis">
      <string>GABOR Filter - angle 1..4 ~ 1*np.pi/angle</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupButtonsBox">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>730</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Images</string>
    </property>
    <widget class="QPushButton" name="addImgButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>180</red>
           <green>146</green>
           <blue>66</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>180</red>
           <green>146</green>
           <blue>66</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>180</red>
           <green>146</green>
           <blue>66</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Image(s)</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveSceneImgButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Scene as Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveImgButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Selected as Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>1471</width>
      <height>900</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>4000</width>
      <height>3000</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QLabel" name="scaleEditLabel">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scale:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scaleBox">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>100</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>1499</x>
      <y>130</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel_2">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>20</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel_3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>60</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clearImgButton">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>690</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clear Image(s)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1871</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionDefault_Values"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDefault_Values">
   <property name="text">
    <string>Default Values</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
